app/components/account/TokenAccountSection.tsx

```diff
@@ -497,3 +552,509 @@ function MultisigAccountCard({ account, info }: { account: Account; info: Multis
+                    <tr>
+                        <td>Non-confidential Credits</td>
+                        <td className="text-lg-end">
+                            {extension.allowNonConfidentialCredits ? 'enabled' : 'disabled'}
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Available Balance</td>
+                        <td className="text-lg-end">{extension.availableBalance}</td>
+                    </tr>
+                    <tr>
+                        <td>Decryptable Available Balance</td>
+                        <td className="text-lg-end">{extension.decryptableAvailableBalance}</td>
+                    </tr>
+                    <tr>
+                        <td>Pending Balance, Low Bits</td>
+                        <td className="text-lg-end">{extension.pendingBalanceLo}</td>
+                    </tr>
+                    <tr>
+                        <td>Pending Balance, High Bits</td>
+                        <td className="text-lg-end">{extension.pendingBalanceHi}</td>
+                    </tr>
+                    <tr>
+                        <td>Pending Balance Credit Counter</td>
+                        <td className="text-lg-end">{extension.pendingBalanceCreditCounter}</td>
+                    </tr>
+                    <tr>
+                        <td>Expected Pending Balance Credit Counter</td>
+                        <td className="text-lg-end">{extension.expectedPendingBalanceCreditCounter}</td>
+                    </tr>
+                    <tr>
+                        <td>Actual Pending Balance Credit Counter</td>
+                        <td className="text-lg-end">{extension.actualPendingBalanceCreditCounter}</td>
+                    </tr>
+                    <tr>
+                        <td>Maximum Pending Balance Credit Counter</td>
+                        <td className="text-lg-end">{extension.maximumPendingBalanceCreditCounter}</td>
+                    </tr>
+                </>
+            );
+        }
+        case 'immutableOwner': {
+            return (
+                <tr>
+                    <td>Immutable Owner</td>
+                    <td className="text-lg-end">enabled</td>
+                </tr>
+            );
+        }
+        case 'memoTransfer': {
+            const extension = create(tokenExtension.state, MemoTransfer);
+            return (
+                <tr>
+                    <td>Require Memo on Incoming Transfers</td>
+                    <td className="text-lg-end">{extension.requireIncomingTransferMemos ? 'enabled' : 'disabled'}</td>
+                </tr>
+            );
+        }
+        case 'transferHookAccount': {
+            const extension = create(tokenExtension.state, TransferHookAccount);
+            return (
+                <tr>
+                    <td>Transfer Hook Account</td>
+                    <td className="text-lg-end">{!extension.transferring && 'not '}transferring</td>
```

Comment:
More of a question, how come this is labelled "Account" when it just seems to be a boolean?

Reply:
Good point, this was a bit lazy. How about "Transfer Hook Status"? It's just a flag saying if the account is in the middle of a transfer or not. So unless there's a big bug, it should always show it's not transferring.

I can just remove showing it totally, but people probably want to see if their account has a transfer hook. Anyway, I changed it to "Transfer Hook Status" in 4b6a068

Reply:
Transfer Hook Status sounds good!